### DISCUSSION TITLE
My code takes too long time.

### DISCUSSION THREAD

**Discussion by phs7646** [ANLAB-KAIST/KENSv3#33] [Project Q&A 1. Application Layer (Socket)]
I finished my code and it passes all test cases. But when compared to document example output, I think it takes too long time. Is it okay or I should fix my code again?

Below is the result.

[==========] Running 15 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 15 tests from EchoTesting
[ RUN      ] EchoTesting.SingleEcho
[       OK ] EchoTesting.SingleEcho (9 ms)
[ RUN      ] EchoTesting.SingleWhoRU
[       OK ] EchoTesting.SingleWhoRU (7 ms)
[ RUN      ] EchoTesting.SingleWhoAmI
[       OK ] EchoTesting.SingleWhoAmI (1749 ms)
[ RUN      ] EchoTesting.SingleHello
[       OK ] EchoTesting.SingleHello (9 ms)
[ RUN      ] EchoTesting.OnetoManyEcho
[       OK ] EchoTesting.OnetoManyEcho (201 ms)
[ RUN      ] EchoTesting.OnetoManyWhoRU
[       OK ] EchoTesting.OnetoManyWhoRU (186 ms)
[ RUN      ] EchoTesting.OnetoManyWhoAmI
[       OK ] EchoTesting.OnetoManyWhoAmI (72465 ms)
[ RUN      ] EchoTesting.OnetoManyHello
[       OK ] EchoTesting.OnetoManyHello (155 ms)
[ RUN      ] EchoTesting.OnetoManyAll
[       OK ] EchoTesting.OnetoManyAll (66700 ms)
[ RUN      ] EchoTesting.ManytoOneEcho
[       OK ] EchoTesting.ManytoOneEcho (53 ms)
[ RUN      ] EchoTesting.ManytoOneWhoRU
[       OK ] EchoTesting.ManytoOneWhoRU (54 ms)
[ RUN      ] EchoTesting.ManytoOneWhoAmI
[       OK ] EchoTesting.ManytoOneWhoAmI (16623 ms)
[ RUN      ] EchoTesting.ManytoOneHello
[       OK ] EchoTesting.ManytoOneHello (61 ms)
[ RUN      ] EchoTesting.ManytoOneAll
[       OK ] EchoTesting.ManytoOneAll (17655 ms)
[ RUN      ] EchoTesting.ManytoMany
[       OK ] EchoTesting.ManytoMany (102832 ms)
[----------] 15 tests from EchoTesting (278766 ms total)

[----------] Global test environment tear-down
[==========] 15 tests from 1 test suite ran. (278766 ms total)
[  PASSED  ] 15 tests.

**Comment by Pusnow** [comment]
We will give enough time and memory for each grading.

It is okay, but 278766 ms seems too big.
